examples/test_data_gen/construct_test_data_flow/flow.dag.yaml

```diff
@@ -1,3 +1,4 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
 id: template_standard_flow
 name: Template Standard Flow
```

Comment:
Traceback (most recent call last):
  File "D:\proj\github\ms\promptflow\examples\test_data_gen\test_data_gen_pipeline\run_test_data_gen_pipeline.py", line 100, in <module>
    pipeline_with_flow = test_data_gen_pipeline_with_flow(
  File "D:\Tools\Anaconda3\envs\prompt-flow\lib\site-packages\azure\ai\ml\dsl\_pipeline_decorator.py", line 203, in wrapper
    pipeline_component = pipeline_builder.build(
  File "D:\Tools\Anaconda3\envs\prompt-flow\lib\site-packages\azure\ai\ml\dsl\_pipeline_component_builder.py", line 187, in build
    outputs, _locals = get_outputs_and_locals(self.func, kwargs)
  File "D:\Tools\Anaconda3\envs\prompt-flow\lib\site-packages\azure\ai\ml\_utils\_func_utils.py", line 468, in get_outputs_and_locals
    return _get_persistent_locals_builder().call(func, _all_kwargs)
  File "D:\Tools\Anaconda3\envs\prompt-flow\lib\site-packages\azure\ai\ml\_utils\_func_utils.py", line 63, in call
    return self._call(func, _all_kwargs)
  File "D:\Tools\Anaconda3\envs\prompt-flow\lib\site-packages\azure\ai\ml\_utils\_func_utils.py", line 109, in _call
    outputs = func(**_all_kwargs)
  File "D:\proj\github\ms\promptflow\examples\test_data_gen\test_data_gen_pipeline\run_test_data_gen_pipeline.py", line 36, in test_data_gen_pipeline_with_flow
    flow_component = load_component(flow_yml_path)
  File "D:\Tools\Anaconda3\envs\prompt-flow\lib\site-packages\azure\ai\ml\entities\_load_functions.py", line 401, in load_component
    component_entity = load_common(Component, source, relative_origin, **kwargs)
  File "D:\Tools\Anaconda3\envs\prompt-flow\lib\site-packages\azure\ai\ml\entities\_load_functions.py", line 125, in load_common
    validation_result.try_raise(error_func=build_error)
  File "D:\Tools\Anaconda3\envs\prompt-flow\lib\site-packages\azure\ai\ml\entities\_validation\core.py", line 252, in try_raise
    raise error_func(
marshmallow.exceptions.ValidationError: Validation for ComponentSchema failed:

 {
  "result": "Failed",
  "errors": [
    {
      "message": "Component name should only contain lower letter, number, underscore and start with a lower letter. Currently got Template Standard Flow.",
      "path": "name",
      "value": "Template Standard Flow",
      "location": "D:\\proj\\github\\ms\\promptflow\\examples\\test_data_gen\\construct_test_data_flow\\flow.dag.yaml#line 3"
    }
  ]
}

Reply:
no id and name in schema, delete them